concepticondata/conceptlists/Pagel-2018-200.tsv

```diff
@@ -0,0 +1,201 @@
+Pagel-2018-200-32	32	dry	1398	DRY	0.000122853
+Pagel-2018-200-33	33	dull			0.000386042
+Pagel-2018-200-34	34	dust	2	DUST	0.000310687
+Pagel-2018-200-35	35	ear	1247	EAR	4.94428E-05
+Pagel-2018-200-36	36	earth			0.000214117
+Pagel-2018-200-37	37	eat	1336	EAT	9.90258E-05
+Pagel-2018-200-38	38	egg	744	EGG	0.000152627
+Pagel-2018-200-39	39	eye	1248	EYE	6.11857E-05
+Pagel-2018-200-40	40	fall	1280	FALL	0.000211854
+Pagel-2018-200-41	41	far	1406	FAR	0.000144701
+Pagel-2018-200-42	42	fat	323	FAT (ORGANIC SUBSTANCE)	0.000373897
+Pagel-2018-200-43	43	father	1217	FATHER	0.000179341
+Pagel-2018-200-44	44	fear	1419	FEAR (BE AFRAID)	0.000233993
+Pagel-2018-200-45	45	feather	1201	FEATHER	0.000164691
+Pagel-2018-200-46	46	few	1242	FEW	0.000209557
+Pagel-2018-200-47	47	fight	1423	FIGHT	0.000360404
+Pagel-2018-200-48	48	fire	221	FIRE	0.000163592
+Pagel-2018-200-49	49	fish	227	FISH	8.1256E-05
+Pagel-2018-200-50	50	five	493	FIVE	4.67E-06
+Pagel-2018-200-51	51	float	1574	FLOAT	0.000349614
+Pagel-2018-200-52	52	flow	2003	FLOW	0.000278764
+Pagel-2018-200-53	53	flower	239	FLOWER	0.000123561
+Pagel-2018-200-54	54	fly			0.00011732
+Pagel-2018-200-55	55	fog	249	FOG	0.000433216
+Pagel-2018-200-56	56	foot	1301	FOOT	7.68081E-05
+Pagel-2018-200-57	57	four	1500	FOUR	1.49124E-05
+Pagel-2018-200-58	58	freeze	1431	FREEZE	0.000174126
+Pagel-2018-200-59	59	fruit	1507	FRUIT	0.000264135
+Pagel-2018-200-60	60	give	1447	GIVE	7.36106E-05
+Pagel-2018-200-61	61	good	1035	GOOD	0.000260401
+Pagel-2018-200-62	62	grass	606	GRASS	0.000169647
+Pagel-2018-200-63	63	green	1425	GREEN	0.000175505
+Pagel-2018-200-64	64	guts	1334	GUTS	0.000352344
+Pagel-2018-200-65	65	hair	1040	HAIR	0.000223414
+Pagel-2018-200-66	66	hand	1277	HAND	5.99757E-05
+Pagel-2018-200-67	67	he	1211	HE	0.000196944
+Pagel-2018-200-68	68	head	1256	HEAD	0.000126058
+Pagel-2018-200-69	69	hear	1408	HEAR	0.000202906
+Pagel-2018-200-70	70	heart	1223	HEART	0.000132626
+Pagel-2018-200-71	71	heavy	1210	HEAVY	0.000188394
+Pagel-2018-200-72	72	here	136	HERE	0.000285826
+Pagel-2018-200-73	73	hit	1433	HIT	0.000281492
+Pagel-2018-200-74	74	hold	1448	HOLD	0.000225
+Pagel-2018-200-75	75	how	1239	HOW	9.09425E-05
+Pagel-2018-200-76	76	hunt	1435	HUNT	0.000243819
+Pagel-2018-200-77	77	husband	1200	HUSBAND	0.000271125
+Pagel-2018-200-78	78	I	1209	I	2.42122E-05
+Pagel-2018-200-79	79	ice	617	ICE	0.000165464
+Pagel-2018-200-80	80	if	1459	IF	0.000328516
+Pagel-2018-200-81	81	in	1460	IN	0.000132633
+Pagel-2018-200-82	82	kill	1417	KILL	0.000231844
+Pagel-2018-200-83	83	know	1410	KNOW (SOMETHING)	0.000154281
+Pagel-2018-200-84	84	lake	624	LAKE	0.000233513
+Pagel-2018-200-85	85	laugh	1355	LAUGH	0.00012791
+Pagel-2018-200-86	86	leaf	628	LEAF	0.000226195
+Pagel-2018-200-87	87	left	244	LEFT	0.000291936
+Pagel-2018-200-88	88	leg	1297	LEG	0.000293917
+Pagel-2018-200-89	89	lie			0.000226318
+Pagel-2018-200-90	90	live	1422	BE ALIVE	7.1041E-05
+Pagel-2018-200-91	91	liver	1224	LIVER	0.000168724
+Pagel-2018-200-92	92	long			0.000127153
```

Comment:
long is not linked?

Reply:
Could check original list, otherwise ambiguous? long (distance), long (duration), ...

Reply:
For consistency, as in all cases, you should check the mother list from which this came. There we usually have the original meaning, and long, for example prototypically points to distance.